File: src/main/java/com/heybys/oddments/fooddelivery/domain/shop/MenuRepository.java
package com.heybys.oddments.fooddelivery.domain.shop;

import java.util.List;

import com.heybys.oddments.base.domain.Repository;

public interface MenuRepository extends Repository<Menu, MenuId> {

    List<Menu> findOpenMenusIn(ShopId shopId);

    List<Menu> find(ShopId shopId);
}

File: src/main/kotlin/com/heybys/oddments/fooddelivery/domain/shop/MenuRepository.kt
package com.heybys.oddments.fooddelivery.domain.shop

import com.heybys.oddments.base.domain.Repository

interface MenuRepository : Repository<Menu, MenuId> {

    fun findOpenMenusIn(shopId: ShopId): List<Menu>

    fun find(shopId: ShopId): List<Menu>
}
